features/all-positions/PositionActionsDialog.tsx

```diff
@@ -0,0 +1,583 @@
+                        rel="noopener noreferrer"
+                      >
+                        UMA docs
+                      </a>
+                      .
+                    </Important>
+                    <Box pt={2} pb={2}>
+                      <Grid container spacing={4}>
+                        <Grid item xs={6}>
+                          <TextField
+                            fullWidth
+                            type="number"
+                            inputProps={{ min: "0" }}
+                            label={`Min collateral/token`}
+                            placeholder="1234"
+                            value={minCollPerToken}
+                            onChange={(
+                              e: React.ChangeEvent<HTMLInputElement>
+                            ) => setMinCollPerToken(e.target.value)}
+                          />
+                        </Grid>
+                        <Grid item xs={6}>
+                          <TextField
+                            fullWidth
+                            type="number"
+                            inputProps={{ min: "0" }}
+                            label={`Max collateral/token`}
+                            placeholder="1234"
+                            value={maxCollPerToken}
+                            onChange={(
+                              e: React.ChangeEvent<HTMLInputElement>
+                            ) => setMaxCollPerToken(e.target.value)}
+                          />
+                        </Grid>
+                        <Grid item xs={6}>
+                          <TextField
+                            fullWidth
+                            type="number"
+                            inputProps={{ min: "0" }}
+                            label={`Max tokens to liquidate`}
+                            placeholder="1234"
+                            error={tokenBalanceTooLow}
+                            helperText={
+                              tokenBalanceTooLow
+                                ? `${tokenSymbol} balance too low`
+                                : null
+                            }
+                            value={maxTokensToLiquidate}
+                            onChange={(
+                              e: React.ChangeEvent<HTMLInputElement>
+                            ) => setMaxTokensToLiquidate(e.target.value)}
+                          />
+                        </Grid>
+                        <Grid item xs={6}>
+                          <FormControl fullWidth>
+                            <InputLabel>Deadline</InputLabel>
+                            <Select
+                              value={deadline}
+                              onChange={(
+                                e: React.ChangeEvent<{ value: unknown }>
+                              ) => setDeadline(e.target.value as string)}
+                            >
+                              <MenuItem value={60}>1 minute</MenuItem>
```

Comment:
Should we even give the user the option to select a deadline? Its a bit unclear what the deadline does/how it works, should we not just default it to 30 mins?

Reply:
yes, I think we should. 30 mins is arbitrary; I'd much rather let the user choose this. They are choosing other complex params in this function, they should also be able to choose this one.